quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/storage/database/Database.java

```diff
@@ -94,7 +94,7 @@ public static boolean isSupported(String alias) {
                 public String apply(String alias) {
                     if ("h2-file".equalsIgnoreCase(alias)) {
                         return "jdbc:h2:file:${kc.home.dir:${kc.db.url.path:~}}" + File.separator + "${kc.data.dir:data}"
-                            + File.separator + "keycloakdb${kc.db.url.properties:;;AUTO_SERVER=TRUE}";
+                            + File.separator + "h2/keycloakdb${kc.db.url.properties:;;AUTO_SERVER=TRUE}";
```

Comment:
@pedroigor as i re-checked I'll also change this line to use File.seperator ;)

Reply:
oops, missed that one :) Thanks.